server/app/views/components/TextFormatter.java

```diff
@@ -184,7 +187,15 @@ private static DivTag buildAccordion(String title, String accordionContent) {
 
   private static UlTag buildList(List<String> items) {
     UlTag listTag = ul().withClasses("list-disc", "mx-8");
-    items.forEach(item -> listTag.with(li().withText(item)));
+    items.forEach(item -> listTag.with(li(applyMarkdown(item))));
     return listTag;
   }
+
+  private static DomContent applyMarkdown(String content) {
+    if (content.charAt(0) == TextFormatter.HEADER) {
```

Comment:
(blocking) Add a guard clause here also to make sure `content` isn't null or empty and that `charAt(0)` can't blow up.

Reply:
done